src/Configuration/Dotenv/Scheme/Parser.hs

```diff
@@ -0,0 +1,37 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+module Configuration.Dotenv.Scheme.Parser where
+
+import Control.Applicative
+import Data.Either
+import Data.Void
+import Text.Megaparsec
+import Text.Megaparsec.Char
+
+import Configuration.Dotenv.Scheme.Helpers
+import Configuration.Dotenv.Scheme.Types
+
+areParseable
+  :: Config
+  -> [(String, String)]
+  -> Bool
```

Comment:
This would be better if you return a `Either e a` where `Left error` is an error message indicating why you could not parse it.